lib/internal/test_runner/reporter/spec.js

```diff
@@ -142,10 +144,12 @@ class SpecReporter extends Transform {
     }
     const results = [`\n${colors['test:fail']}${symbols['test:fail']}failing tests:${white}\n`];
     for (let i = 0; i < this.#failedTests.length; i++) {
-      ArrayPrototypePush(results, this.#formatTestReport(
-        'test:fail',
-        this.#failedTests[i],
-      ));
+      const test = this.#failedTests[i];
+      const relPath = relative(this.#cwd, test.file);
+      const formattedErr = this.#formatTestReport('test:fail', test);
+      const location = `test at line ${test.line}, column ${test.column} of '${relPath}'`;
```

Comment:
```suggestion
      const location = `failure at line ${test.line}, column ${test.column} of '${relPath}'`;
```

Reply:
I specifically used "test" instead of "failure" because I didn't want people to confuse the test's location with the error's location.